boards/sensry/ganymed_bob/Kconfig.defconfig

```
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 sensry.io
+# SPDX-License-Identifier: Apache-2.0
+
+if NETWORKING
+
+config NET_L2_ETHERNET
+	default y
```

```suggestion
configdefault NET_L2_ETHERNET
	default y
```

see https://docs.zephyrproject.org/latest/build/kconfig/extensions.html#kconfig-extensions
so we don't need to add the other dependencies here too

---

that seems to be correct. NET_L2_ETHERNET will not be included once NETWORKING is not selected. 

however, there are very many board doing it the same way ;)